src/main/java/com/ft/methodearticleinternalcomponentsmapper/health/RemoteServiceHealthCheck.java

```diff
@@ -54,7 +55,11 @@ public RemoteServiceHealthCheck(
     public AdvancedResult checkAdvanced() throws Exception {
         ClientResponse response = null;
         try {
-            response = client.resource(remoteServiceUri).header("Host", hostHeader).get(ClientResponse.class);
+            WebResource.Builder builder = client.resource(remoteServiceUri).getRequestBuilder();
+            if (hostHeader != null && !hostHeader.isEmpty()) {
```

Comment:
Use Strings.isNullOrEmpty, you used it in other files.

Reply:
:) https://github.com/Financial-Times/methode-article-internal-components-mapper/pull/26/commits/ee8cf352b07b18ef70347865b6330f4d3d9483b7
I told him not to use, it brings in a dependency. I know we already have it via maven, but i'm trying to go in the direction of getting rid rather than expanding on it.

Reply:
ok, good